CMakeLists.txt

```diff
@@ -442,6 +442,13 @@ add_flex_bison_dependency(esql_lexer esql_grammar)
 add_custom_target(gen_esql_grammar DEPENDS ${BISON_esql_grammar_OUTPUTS})
 add_custom_target(gen_esql_lexer DEPENDS ${FLEX_esql_lexer_OUTPUTS})
 
+# For java stored procedure server
```

Comment:
Maven Central Repository must be connected with **TLS 1.2**+ (Since January 15, 2020). If not, SSL Handshake Execption occurs, it means ivy download will fail and eventually build will be failed. For this reason, it is necessary to check if the jdk supports TLS 1.2 (6u121+, 7u95+), for example: 

`find_package(Java 1.7.0.95 REQUIRED)`

Reference: [[eGovFrame Technical Report]](https://www.egovframe.go.kr/home/faqinfo/faqinfoRead.do?menuNo=68&faqId=FAQ_0000000000000761)

Reply:
Thank you for the information. I'm going to consider not to get ivy from the Maven Central Repository if downloading is failed.

Reply:
I changed an approach to download ivy.jar from CMake instead of taking a limitation depending on the Java version at 787d05a